modules/rosa-mobb-prereq-checklist.adoc

```diff
@@ -0,0 +1,198 @@
+
+=== AWS CLI (`aws`)
+
+* Install from https://aws.amazon.com/cli/[AWS Command Line Interface] if you have not already.
+* Configure the CLI:
++
+. Enter `aws configure` in the terminal: 
++
+[source,terminal]
+----
+$ aws configure
+----
++
+. Enter the AWS Access Key ID and press *enter*.
+. Enter the AWS Secret Access Key and press *enter*.
+. Enter the default region you want to deploy into.
+. Enter the output format you want, “table” or “json”. 
+. Verify the output by running:
++
+[source,terminal]
+----
+ $ aws sts get-caller-identity
+----
++
+. Ensure that the service role for ELB already exists by running:
++
+[source,terminal]
+----
+$ aws iam get-role --role-name "AWSServiceRoleForElasticLoadBalancing"
+----
++
+.. If it does not exist, run:
++
+[source,terminal]
+----
+$ aws iam create-service-linked-role --aws-service-name "elasticloadbalancing.amazonaws.com"
+----
+
+=== Red Hat account
+
+* Create a https://console.redhat.com/[hybrid-console]  account if you have not already.
+
+=== ROSA CLI (`rosa`)
+
+. Enable ROSA from your AWS account on the https://console.aws.amazon.com/rosa/[AWS console] if you have not already.
+. Install the CLI from https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html[Installing the Red Hat OpenShift Service on AWS (ROSA) CLI, rosa[] or from the OpenShift console https://console.redhat.com/openshift/downloads#tool-rosa[AWS console].
+. Enter `rosa login` in a terminal, and this will prompt you to go to the https://console.redhat.com/openshift/token/rosa[token page] through the console:
++
+[source,terminal]
+----
+$ rosa login
+----
++
+. Log in with your Red Hat account credentials.
+. Click the *Load token* button.
+. Copy the token and paste it back into the CLI prompt and press *enter*.
++
+* Alternatively, you can copy the full `$ rosa login --token=abc...` command and paste that in the terminal:
++
+[source,terminal]
+----
+$ $ rosa login --token=<abc..>
+----
++
+. Verify your credentials by running:
++
+[source,terminal]
+----
+$ rosa whoami
+----
++
+. Ensure you have sufficient quota by running: 
++
+[source,terminal]
+----
+rosa verify quota
+----
++
+* See link:https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html#rosa-aws-policy-provisioned_rosa-sts-aws-prereqs[Provisioned AWS Infrastructure] for more details on AWS services provisioned for ROSA cluster. 
+* See link:https://docs.openshift.com/rosa/rosa_planning/rosa-sts-required-aws-service-quotas.html[Required AWS service quotas] for more details on AWS services quota. 
+
+=== OpenShift CLI (`oc`)
+
+. Install from [here](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html) or from the OpenShift console [here](https://console.redhat.com/openshift/downloads#tool-oc).
```

Comment:
These links are still in Markdown. Should be:

* link:https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html[here]
* link:https://console.redhat.com/openshift/downloads#tool-oc[here]